admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -131,7 +131,7 @@
 
       <tbody class="bg-white text-3.5 line-[150%] text-black">
         <% @rows.each do |row| %>
-          <tr class="<%= row_class_for(row) %>">
+          <tr class="<%= row_class_for(row) %>" data-action="click-><%= stimulus_id %>#toggleRowSelection">
```

Comment:
For non-standard elements like `<tr>` we must specify the event `click->` in `data-action`.